content/html/concepts/tags/terms/img/img.md

```diff
@@ -1,30 +1,31 @@
 ---
 Title: '<img>'
-Description: 'Used to add an image to the document. It can also be used inside an anchor tag to create an image link.'
+Description: 'Displays an image on the web page.'
 Subjects:
   - 'Web Development'
   - 'Web Design'
 Tags:
+  - 'Images'
   - 'Tags'
 CatalogContent:
   - 'learn-html'
   - 'paths/front-end-engineer-career-path'
 ---
 
-Used to add an image to the document. It can also be used inside an anchor tag to create an image link.
+The `<img>` tag displays an image on the web page.
 
 ## Syntax
 
 ```html
 <img src="image URL" alt="text used for accessibility" />
 ```
 
-`<img>` is a tag that has no closing tag, meaning that it cannot have tags inside of it. It uses attributes to set properties, some common attributes are:
+The HTML image tag, `<img>`, has no closing tag, meaning that it cannot have other tags inside of an image element. It uses attributes to set properties, some common attributes are:
```

Comment:
```suggestion
The HTML image tag, `<img>`, has no closing tag. An image element cannot have other tags inside of it. It uses attributes to set properties, some common attributes are:
```

After reading over this again, one last thing ☝️

Reply:
@ethan-master-coding Also, go ahead and update the branch! Going to merge this after